src/misc/utils.js

```diff
@@ -6,6 +6,14 @@ const isAbsolutePath = path.isAbsolute;
 // Based on https://github.com/isaacs/node-glob/blob/master/glob.js#L83
 // (glob.hasMagic)
 export function hasMagic(pattern, options) {
+
+  //support RegExp as well as glob
```

Comment:
`// Support` and remove empty line between function name and the comment please.